src/components/AddBook.js

```diff
@@ -1,14 +1,68 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
 
-const AddBook = () => (
-  <div>
-    <h3>Add new book</h3>
-    <form>
-      <input type="text" id="title" placeholder="Book Title" />
-      <input type="text" id="author" placeholder="Author" />
-      <button type="button">Add Book</button>
-    </form>
-  </div>
-);
+const newBookState = {
+  title: '',
+  author: '',
+};
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+  const [book, changeBook] = React.useState(newBookState);
```

Comment:
- [ ] OPTIONAL: You can import the useState hook together with react:
`import React, { useState } from 'react';`
 and use it directly: 
`const [book, changeBook] = useState(newBookState);`.

Reply:
@ShoiraTa 
Thanks. I didn't noticed this before. I'll make the change.